scripts/src/chartprreview/chartprreview.py

```diff
@@ -43,16 +50,18 @@ def check_owners_file_against_directory_structure(username, category, organizati
     vendor_label = out["vendor"]["label"]
     chart_name = out["chart"]["name"]
     error_exit = False
+    msgs = []
     if organization != vendor_label:
         error_exit = True
-        print("vendor/label in OWNERS file doesn't match the directory structure.")
+        msg.append("[ERROR] vendor/label in OWNERS file doesn't match the directory structure")
```

Comment:
Maybe  vendor/label in OWNERS file (${vendor_label} doesn't match the directory structure (${organization})

or other way if including the expected and sbmitted values.

Same for chart name message below.

Reply:
We should tell use how this structure should look like.

Reply:
Done. 229bd4e5c3b304f1c759c7b5b8f51971defae155